tests/unittests/test_net.py

```diff
@@ -1347,6 +1347,49 @@
             """),
         },
     },
+    'wakeonlan': {
+        'expected_eni': textwrap.dedent("""\
+            auto lo
+            iface lo inet loopback
+
+            auto iface0
+            iface iface0 inet dhcp
+                ethernet-wol g
+        """).rstrip(' '),
+        'expected_netplan': textwrap.dedent("""
+            network:
+                ethernets:
+                    iface0:
+                        dhcp4: true
+                        wakeonlan: true
+                version: 2
+        """).rstrip(' '),
```

Comment:
dedent is weird sometimes.

I don't htink you should need the rstrip here, and think you might have added it in an attempt to combat misuse of dedent.

You really need the \ on line 1359.  dedent will remove the same number of spaces from the beginning of each line. but in this stanza, you your first line has no indentation (it is just "\n").  Add the \ and I think you can drop the rstrip.

Reply:
I'd copied (and modified) this testcase structure from "dhcpv6_stateless" and so that's inherited from it.

I'll test your suggestions and update accordingly.

Reply:
I've removed the rstrip.

However if I add the trailing "\" to line 1359 the testcase fails - seems there needs to be a blank line before the "network:". There are other testcases in this file that are the same (I copied from one of them).